Dockerfile

```diff
@@ -29,11 +29,14 @@ RUN node_modules/.bin/esbuild --bundle --outdir=public "application.ts" "modules
     --loader:.html=copy --loader:.png=copy \
     --loader:.ttf=file --loader:.eot=file --loader:.woff=file --loader:.svg=file --loader:.woff2=file
 
+# Build backend
+RUN node esbuild.config.mjs build
```

Comment:
this feels much cleaner than the garbage multiline esbuild command i made

Reply:
I liked the CLI option better actually haha.

But it should be possible to combine both frontend build and backend build together. I was thinking of doing it but wanted to change less things.